content/api/motor-board.md

```diff
@@ -72,6 +72,5 @@ r.motor_board.m1 = COAST
 ```
 
 {{% notice warning %}}
-Changing the motor speed by more than 1 in a single operation (e.g. `-1` to `0`, `1` to `-1` etc) will likely trigger the current protection and your robot will shut down with a distinct beeping noise.
-{{% /notice %}} 
-
+Changing the motor speed by more than 1 in a single operation (e.g. `-1` to `0`, `1` to `-1` etc.) will likely trigger the current protection and your robot will shut down with a distinct beeping noise.
```

Comment:
The first example change (`-1` to `0`) seems wrong -- could we either explain why it's right or fix it?

Reply:
Please could you clarify what you mean by this? Are you referring to the fact that the difference between `-1` and `0` is not strictly "more than 1"?

Reply:
Yes: the difference between -1 and 0 is 1, which is not "more than 1".

Reply:
I see. As far I can tell the current threshold of 1 is completely arbitrary, so how about we change this to "Large changes in the motor speed (e.g. `-1` to `0`, `1` to `-1` etc.)..."?

Perhaps we could recommend that the motor speed be changed only in small increments as well, so that users have examples of both "what is bad" and "what is good".

Reply:
addressed in 71d0b125be69ec2f51083e2da1c1ca907dffa9c4